### PR TITLE
Add restart button

### LINKED ISSUE #69: Means to restart kernel
I am testing thebelab with some C++ kernels for which restarts are more frequent, and that would be a great usability improvements to have a means to restart kernels.

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Fixes #69 

Seems to work fine. (Adding a restart button next to the run.)

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
ping @minrk.

adds a `restart` button next to the `run`.

**Comment by SylvainCorlay** [comment]
@minrk when this gets in, I will add a C++ example!

**Comment by SylvainCorlay** [comment]
up?

File: src/thebelab.js [hunk 1]
```diff
@@ -167,6 +167,11 @@ function renderCell(element, options) {
       .text("run")
       .click(execute)
   );
+  $cell.append(
+    $("<button class='thebelab-button thebelab-restart-button'>")
+      .text("restart")
```
Comment (minrk) [inline src/thebelab.js]: Can you add `.attr('title', 'restart the kernel')` so there's info about what it does? Maybe a similar 'run this cell' title on the run button while we're at it.

Comment (SylvainCorlay) [reply src/thebelab.js]: Done.

File: src/thebelab.js [hunk 2]
```diff
@@ -164,9 +164,14 @@ function renderCell(element, options) {
   $cell.append($cm_element);
   $cell.append(
     $("<button class='thebelab-button thebelab-run-button'>")
-      .text("run")
+      .text("run this cell")
       .click(execute)
   );
+  $cell.append(
+    $("<button class='thebelab-button thebelab-restart-button'>")
+      .text("restart the kernel")
```
Comment (minrk) [inline src/thebelab.js]: Sorry, not `.text`, which changes the button label, but `.attr("title", "restart the kernel")` to give hover text. Leave the button text as "run" and "restart"

**Pull request merged**